Course/Course/Presentation/Handouts/HandoutsUpdatesDetailView.swift

```diff
@@ -27,12 +27,16 @@ public struct HandoutsUpdatesDetailView: View {
         router: CourseRouter,
         cssInjector: CSSInjector
     ) {
-        if handouts != nil {
+        let handoutsView = handouts != nil && announcements == nil
+        let noHandouts = handouts == nil && announcements == nil
+        
+        if handoutsView || noHandouts {
```

Comment:
it appears than this condition can be replaced just with `announcements == nil`

Reply:
Done